tests-utils/src/main/scala/tech/beshu/ror/utils/containers/images/ReadonlyRestPlugin.scala

```diff
@@ -60,20 +61,29 @@ object ReadonlyRestPlugin {
   }
 }
 class ReadonlyRestPlugin(esVersion: String,
-                         config: Config)
+                         config: Config,
+                         performInstallation: Boolean)
   extends Elasticsearch.Plugin {
 
   override def updateEsImage(image: DockerImageDescription): DockerImageDescription = {
-    image
+    val withoutInstallation = image
```

Comment:
Oh, I get it now. I thought we were disabling the patching, not the ROR installation itself. 
It would be good if we could allow ES to install ROR and disable patching. I guess the problem is that the ES doesn't start (because ROR makes it explode, right?). 

When you run ES docker from console you can override its entrypoint (`docker run -ti docker.elastic.co/elasticsearch/elasticsearch:9.0.0-beta1 bash`). Are we able to sth similar with test-containers?

Reply:
Done, with some tweaks (`/bin/sh -c "while true; do sleep 30; done`)
- in order to keep the container alive
- also I had to modify checking container readiness - it depended on ES healthcheck endpoint